content/examples/databricks-cluster.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Using sparklyr with Databricks"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+This documentation demonstrates how to use `sparklyr` with RStudio Server Pro
+and a Databricks cluster with Apache Spark.
+
+## Best practices
+
+- **Connect to Spark remotely** - Install RStudio Server Pro outside of the
+  Databricks cluster and work with Spark remotely
+- **Work across RStudio Team** - Install RStudio Connect and RStudio Package
+  Manager outside of the Databricks cluster so that they are not limited to the
+  resources or ephemeral nature of Databricks clusters
```

Comment:
I feel like these two could be condensed into a single point: Install all your stuff outside of Databricks.

Reply:
I reworked these items in 1ca22c5. Although I still ended up with two points important enough to call them out separately, 1) install RSP/RSC/RSPM outside of Databricks, 2) work with Databricks remotely.